src/hotspot/share/adlc/formsopt.cpp

```diff
@@ -365,14 +365,14 @@ void RegClass::build_register_masks(FILE* fp) {
   for(i = 0; i < len - 1; i++) {
     fprintf(fp," 0x%x,", regs_in_word(i, false));
   }
-  fprintf(fp," 0x%x );\n", regs_in_word(i, false));
```

Comment:
Does use of RegMask_Size() above for the length still make sense?  IIUC, RegMask_Size() is what previously determined the max size of frames and register masks.

Reply:
`RegMask_Size()` [determines](https://github.com/dlunde/jdk/blob/953966686ef7b86972bb2c156c931c60c4e5cb70/src/hotspot/share/adlc/output_h.cpp#L101) the base, statically allocated, chunk for register masks. If I'm not mistaken, it does not affect max frame size. The "all-stack" bit of current register masks already allows frame growth beyond what a register mask can represent.